image_proc/launch/image_proc.launch

```diff
@@ -1,29 +1,64 @@
+<?xml version="1.0"?>
 <!-- Launch in the camera namespace containing "image_raw" and "camera_info" -->
 <launch>
+  <!-- Manager name must be globally qualified -->
+  <arg name="manager" /> 
+  <!-- Worker threads -->
+  <arg name="num_worker_threads" default="4"/>
 
-  <arg name="manager" /> <!-- Must be globally qualified -->
-  <arg name="respawn" default="false" />
-  <!-- TODO Arguments for debayer, interpolation methods? -->
+  <arg name="respawn"               default="false" />
+  <arg name="debayer"               default="true" />
```

Comment:
What does `debayer` do?

Reply:
I have left the parameters values such that it is set to whatever it was there previously without using image_resize. I hope it helps keep the package backwards compatible.